.pipelines/pipeline.yaml

```diff
@@ -181,6 +181,7 @@ stages:
     dependsOn:
       - setup
       - containerize
+      - build_windows_images
```

Comment:
please work Windows into the existing pattern (using things like template variables) instead of adding special-case complexity

Reply:
what I mean is if, for example, `containers/container-template.yaml` is not generic enough to support Windows builds directly, maybe it should be `containers/container-template-$(os).yaml`. At that point we have to make no other structural changes to the main pipeline and only need to create that specific step template.

Reply:
I'm still confused, borrowing the existing windows-image.yaml (build_windows_images) and then waiting for it to complete before creating the multi arch manifests here.

Reply:
the `containerize` stage has been set up generically with parameters passed in to build the whole matrix of arch x image that we ship. instead of having Windows separate, fold it in so that `containerize` builds the matrix of os x arch x image.

Reply:
I thought the windows images had to be done from windows pool though

Reply:
https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema/pool?view=azure-pipelines
> You can specify a pool at the pipeline, stage, or job level.